src/components/Home.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import '../App.css';
+import { useSelector, useDispatch } from 'react-redux';
+import { getData, setCountry } from '../redux/reducer';
+import Card from './Card';
+
+const Home = () => {
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that each coin list on the _Homepage_ / _navigation arrow icons_ as well as the _search icon_ are behaving as clickable objects when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to this object's CSS property on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.